render.go

```diff
@@ -168,6 +173,7 @@ func NewRenderer(opts Options) *Renderer {
 		tctxAliases: tctxAliases,
 		lDelim:      opts.LDelim,
 		rDelim:      opts.RDelim,
+		missingKey:  opts.MissingKey,
```

Comment:
this needs to default to `error` if it's empty, rather than sending an empty string

Reply:
Done